test/e2e/upgrade/alert/alert.go

```diff
@@ -96,6 +96,11 @@ func (t *UpgradeTest) Test(f *framework.Framework, done <-chan struct{}, upgrade
 				return framework.ProviderIs("gce")
 			},
 		},
+		{
+			// Should be removed one release after the attached bugzilla is fixed.
+			Selector: map[string]string{"alertname": "HighlyAvailableWorkloadIncorrectlySpread"},
```

Comment:
Can you scope it to the component that is impacted (add an additional label)?  Or does the alert not have an extra label?  A namespace?

Reply:
Good point, I scoped them to the namespace/workload.

Since we have now have two different alerts allowed, I attached the relevant bugzilla to each of them.